src/bin/lib/render-plan.ts

```diff
@@ -0,0 +1,24 @@
+import chalk from 'chalk'
+import { renderRuntimeErrorsAndWriteToFile, renderValidationErrors } from './plan-messages'
+import renderMessages from './plan-messages'
+import { RequestBatch } from '../../lib/offline-api/index'
+
+const renderPlan = async (batches: RequestBatch[], errorsFile: string): Promise<void> => {
```

Comment:
This function returns `undefined` (or `void`) instead of `Promise`, right?

Reply:
It's an async function, so it returns a promise;)

Reply:
You're right. Damm `asyn/await`